test/api.js

```diff
@@ -402,6 +387,42 @@ tape('findOrCreate() recps', (t) => {
   })
 })
 
+test('v1.findOrCreate', (t) => {
+  const dir = '/tmp/metafeeds-metafeed-v1-shard'
+  rimraf.sync(dir)
+  const keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+  const sbot = createSbot({ keys, path: dir })
+
+  const details = {
+    feedpurpose: 'chess',
+    // feedformat: 'classic', optional
+  }
+  sbot.metafeeds.v1.findOrCreate(details, (err, feed) => {
+    if (err) throw err
+    t.equal(feed.feedpurpose, details.feedpurpose, 'creates feed')
+
+    sbot.metafeeds.v1.findOrCreate(details, (_, feed2) => {
+      t.deepEqual(feed, feed2, 'finds feed')
+
+      pull(
+        sbot.metafeeds.branchStream({ root: null, old: true, live: false }),
```

Comment:
:fire: I may not have understood, but this is returning `[]` while the tests before it pass.... feeds are being created.

HALP!

Reply:
I'm not sure I understood, can you clarify?